app/lib/import_export.rb

```diff
@@ -142,6 +142,9 @@ def xlsx_export(config, patients, outer_batch_size, inner_batch_size)
           files << save_file(config, file)
         end
       else
+        # Do not create a file for an export that only has headers.
+        next if workbook.instance_variable_get('@sheets').all? { |sheet| sheet.last_row_number.zero? }
```

Comment:
I'm seeing an issue where if I do custom export, I seem to always get the "no monitorees or monitoree data matched the selected export criteria" page. From what I can tell it is because on this line, every `last_row_number` is `0` no matter what. I see that on this [line](https://github.com/SaraAlert/SaraAlert/blob/df9950e99100d818f2e6ab09614d1aa6b8ad6b4c/app/lib/import_export.rb#L126) @timwongj left a comment explaining that he is tracking `last_row_number` manually because it is not updating. Maybe Tim has some more insight here, is that just a bug in `fast_excel` or something?

Reply:
I updated the logic to be `next if last_row_nums.values.all? { |v| v == 0 }` please tell me if that fixes your test case.

Reply:
Yeah this fixes custom export, but looks like rubocop wants it to be
```ruby
next if last_row_nums.values.all? { |v| v.zero? }
```